content/en/users/Rucio/Getting_started/_index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Getting Started"
+type: docs
+linkTitle: "Rucio Getting started"
+weight: 100
+description: >-
+     How to get started with Rucio as a new user or VO.
+---
+
+## Getting Started
+
+### Getting Started as a new user
+
+1. To get set up with a Rucio account please contact our [support email](mailto:Rucio-Support@stfc365.onmicrosoft.com):
```

Comment:
can we suggest to use GGUS for this requests, so that we are tracked? GGUS -> Rucio SU

Reply:
Addressed in latest push